app/views/shared/_user_mgt_buttons.html.slim

```diff
@@ -0,0 +1,12 @@
+- if current_user.admin? || current_user.id == locals[:user].id
+  = button_to "Edit Info for #{user_path(locals[:user].id)} (#{locals[:user].tone_name})",
```

Comment:
My typical case is I don't like using `button_to` unless I have something that needs a post specifically. For a get, I typically use `link_to` and a class or style that replicates the button formatting.